### PR TITLE
feature(https): Https prepare

### PR DESCRIPTION
**Pull request by Nailykk**
Change env variable in order to support https proxy by only changing theses variable

### CODE REVIEW

**Review by guyts96 — APPROVED** [review]

File: client/src/managers/socketManager.ts [hunk 1]
```diff
@@ -32,7 +32,10 @@ class SocketManager {
     private socket: any;
 
     connect() {
-        this.socket = io(config.BASE_URL);
+    this.socket = io(config.BASE_URL, {
```
Comment (Lukyio) [inline client/src/managers/socketManager.ts]: Needed to be indented

File: client/src/managers/socketManager.ts [hunk 2]
```diff
@@ -32,7 +32,10 @@ class SocketManager {
...
+    	path: config.WS_PATH
+    });
+        console.log(this.socket);
```
Comment (Lukyio) [inline client/src/managers/socketManager.ts]: I'm scared of too much logs

Comment (guyts96) [inline client/src/managers/socketManager.ts]:
```suggestion
```

File: client/src/managers/socketManager.ts [hunk 3]
```diff
@@ -32,7 +32,9 @@ class SocketManager {
     private socket: any;
 
     connect() {
-        this.socket = io(config.BASE_URL);
+    this.socket = io(config.BASE_URL, {
+    	path: config.WS_PATH
+    });
```
Comment (Lukyio) [inline client/src/managers/socketManager.ts]:
```suggestion
        this.socket = io(config.BASE_URL, {
    	    path: config.WS_PATH
        });
```

**Review by Lukyio — APPROVED** [review]

**Pull request merged**